tests/recorded/comprehensions/C413.txt

```diff
@@ -21,16 +22,17 @@ sorted(frozenset('abc'))
 
 # C413: Unnecessary <list/reversed> call around sorted().
 # It’s unnecessary to use list() around sorted()
+# Several of these cases were not handled by shed codemods, but are by ruff
 
 sorted([2, 3, 1])
-reversed(sorted([2, 3, 1]))
-reversed(sorted([2, 3, 1], reverse=True))
+sorted([2, 3, 1], reverse=True)
+sorted([2, 3, 1], reverse=False)
 
 sorted([2, 3, 1], reverse=False)
-reversed(sorted([2, 3, 1], reverse=False))
-reversed(sorted([2, 3, 1], reverse="abc"))
-reversed(sorted([2, 3, 1], reverse=1 + 2 + c))
-reversed(sorted([2, 3, 1], reverse=1 or 2))
+sorted([2, 3, 1], reverse=True)
+sorted([2, 3, 1], reverse=not "abc")
+sorted([2, 3, 1], reverse=not 1 + 2 + c)
+sorted([2, 3, 1], reverse=not 1 or 2)
```

Comment:
Hmmm... looks like this can change semantics due to [operator precedence](https://docs.python.org/3/reference/expressions.html#operator-precedence) sometimes.  Mind working out whether that's `shed` or `ruff`, and reporting as appropriate?

Reply:
`shed` didn't dare messing with the reverse key previously. Opened ruff issue: https://github.com/astral-sh/ruff/issues/10335

I can reinstate `shed`s C413 fixer, or we hope for a timely fix.

Reply:
Happy to leave this as a `ruff` problem, they do fix things quickly.

Reply:
And indeed it was!